scripts/scil_image_math.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+import os
+
+from dipy.io.utils import is_header_compatible
+import nibabel as nib
+import numpy as np
+
+from scilpy.image.operations import (is_float, absolute_value, around,
+                                     addition, ceil, closing, convert,
+                                     difference, dilation, division, erosion,
+                                     floor, gaussian_blur, intersection,
+                                     invert, lower_clip, lower_threshold, mean,
+                                     multiplication, normalize_max,
+                                     normalize_sum, opening, std, subtraction,
+                                     union, upper_threshold, upper_clip,
+                                     get_operation_doc)
+from scilpy.io.utils import (add_overwrite_arg,
+                             add_verbose_arg,
+                             assert_outputs_exist)
+
+OPERATIONS = {
+    'lower_threshold': lower_threshold,
+    'upper_threshold': upper_threshold,
+    'lower_clip': lower_clip,
+    'upper_clip': upper_clip,
+    'absolute_value': absolute_value,
+    'round': around,
+    'ceil': ceil,
+    'floor': floor,
+    'normalize_sum': normalize_sum,
+    'normalize_max': normalize_max,
+    'convert': convert,
+    'invert': invert,
+    'addition': addition,
+    'subtraction': subtraction,
+    'multiplication': multiplication,
+    'division': division,
+    'mean': mean,
+    'std': std,
+    'union': union,
+    'intersection': intersection,
+    'difference': difference,
+    'dilation': dilation,
+    'erosion': erosion,
+    'closing': closing,
+    'opening': opening,
+    'blur': gaussian_blur
+}
+
+DESCRIPTION = """
+Performs an operation on a list of mask images. The supported
+operations are:
+
+Some operations such as multiplication or addition accept float value as
+parameters instead of images.
+> scil_image_math.py multiplication img.nii.gz 10 mult_10.nii.gz
+"""
+
+DESCRIPTION += get_operation_doc()
+
+
+def build_args_parser():
```

Comment:
_build_args_parser

Reply:
The docstring, even the very simple numpy wrapper ?
I would rather not explain ceil, floor, round, etc....